tests/src/test/spark321/scala/org/apache/spark/sql/rapids/RapidsShuffleThreadedReaderSuite.scala

```diff
@@ -113,7 +113,7 @@ class RapidsShuffleThreadedReaderSuite
       val shuffleId = 22
       val numMaps = 6
       val keyValuePairsPerMap = 10
-      val serializer = new GpuColumnarBatchSerializer(NoopMetric)
+      val serializer = new GpuColumnarBatchSerializer(NoopMetric, Array.empty, false)
```

Comment:
Do not we need tests for kudo ?

Reply:
I mean unit tests here. Not integration tests.

Reply:
> See this https://github.com/NVIDIA/spark-rapids/pull/11724#discussion_r1843509785
Not enough. We still need integration tests for both `JCudfSerialization` and `Kudo` until JCudfSerialization is totally dropped.